Makefile

```diff
@@ -111,12 +118,20 @@ generate: vendor controller-gen ## Generates types deepcopy code
 manifests: controller-gen kustomize ## Generates the manifests in $PROJECT_DIR/install
 	controller-gen crd:crdVersions=v1 rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=install/crd output:rbac:artifacts:config=install/rbac && $(KUSTOMIZE) build install > $(AUTHORINO_MANIFESTS)
 	$(MAKE) patch-webhook
-
+	
+run:$(shell $(PROJECT_DIR)/hack/check-version.sh)
+run:version=$(shell $(YQ) '.build.version' build.yaml)
+run:git_sha=$(shell git rev-parse HEAD)
+run:dirty=$(shell $(PROJECT_DIR)/hack/check-git-dirty.sh || echo "unknown")
 run: generate manifests ## Runs the application against the Kubernetes cluster configured in ~/.kube/config
-	go run -ldflags "-X main.version=$(VERSION)" ./main.go server
+	go run -ldflags "-X main.version=$(version) -X main.gitSHA=${git_sha} -X main.dirty=${dirty}" ./main.go server
 
+build:$(shell $(PROJECT_DIR)/hack/check-version.sh)
```

Comment:
I've tried `make build` locally, but got:

```
make: /bin/yq: Command not found
```

We probably need to add `yq` as a dependency of the target. Same for `run` and `docker-build`.

Reply:
Actually the error seems to be `go-install-tool` instead of `go-get-tool`.

Reply:
Hmm. Both things are needed: `go-get-tool` and adding the dependency here.